src/client/utils.ts

```diff
@@ -1,18 +1,29 @@
-import { Range, TextDocument, window, workspace } from 'vscode';
+import {
+	Range,
+	TextDocument,
+	workspace,
+	Uri,
+	WorkspaceConfiguration
+} from "vscode";
 
-const defaultCommand = 'v';
+const defaultCommand = "v";
 
 export function fullDocumentRange(document: TextDocument): Range {
 	const lastLineId = document.lineCount - 1;
 	return new Range(0, 0, lastLineId, document.lineAt(lastLineId).text.length);
 }
 
 export function getVExecCommand(args: string): string {
-	const config = workspace.getConfiguration('v');
-	const vPath = config.get('pathToExecutableFile', '') || defaultCommand;
+	const config = getVConfig();
+	const vPath = config.get("pathToExecutableFile", "") || defaultCommand;
 	return `${vPath} ${args}`;
 }
 
+export function getVConfig(uri?: Uri): WorkspaceConfiguration {
+	if (uri) return workspace.getConfiguration("v", uri);
+	return workspace.getConfiguration("v", null);
```

Comment:
Can be simplified to `return workspace.getConfiguration("v", uri || null);`

Reply:
Yeah... you're right  👍